depfu/bot/lib/depfu.rb

```diff
@@ -14,7 +14,12 @@ def parse
         next unless table_line?(line)
         splitted = line.split('|')
         if ['updated', 'created', 'added'].include?(splitted[1].strip)
+          # add the dependencies (second table)
           result << Gem.new(splitted[2].strip, splitted[4].strip)
+        elsif !['removed'].include?(splitted[1].strip)
```

Comment:
why not just

```ruby
elsif splitted[1].strip != 'removed'
```

:confused:

Reply:
To be consistent with the if :thinking: 
But I don't care too much tbh... We will retire this bot hopefully in a few weeks anyway so I wouldn't put too much effort into it. I can change it if you want though...

Reply:
but in the if you had an array... here you only have one value... :confused:

Reply:
Updated